### PR TITLE
update data commons url to new S3 bucket

### PR DESCRIPTION
**Pull request by catherinebirney**
- updates data commons URL
- pulls parquets/metadata based on max version number instead of upload date. This change implemented as all files moved from old to new S3 bucket have the same upload date/time

Cannot merge this PR until v1.3.0 FLOWSA files are uploaded to the new URL, which is not currently possible as new Data Commons has a max zip file upload of 30 MB and some of our files are 125MB.

### CODE REVIEW

**Comment by bl-young** [comment]
> Cannot merge this PR until v1.3.0 FLOWSA files are uploaded to the new URL, which is not currently possible as new Data Commons has a max zip file upload of 30 MB and some of our files are 125MB.

:cry:

File: esupy/processed_data_mgmt.py [hunk 1]
```diff
@@ -88,7 +89,10 @@ def download_from_remote(file_meta, paths, **kwargs):
                                           + '/' + subdirectory)
                 file = folder + "/" + f
                 create_paths_if_missing(file)
-                log.info('%s saved to %s', f, folder)
+                log.info('%s downloaded from '
+                         'https://dmap-data-commons-ord.s3.amazonaws.com/'
+                         'index.html?prefix=flowsa/ and saved to %s',
+                         f, folder)
```
Comment (bl-young) [inline esupy/processed_data_mgmt.py:95]: this shouldn't be flowsa specific

Comment (catherinebirney) [reply esupy/processed_data_mgmt.py]: Thanks! Fixed it [ae2e1a0](https://github.com/USEPA/esupy/commit/ae2e1a018605ccb650280a0a1b9c79ed14c66bf6)

**Comment by catherinebirney** [comment]
@WesIngwersen @bl-young Nearly all the v1.3.0 FBA/FBS parquets are on the new URL. I'm working with Vincent to get the last few up. All of the parquets we need are uploaded. If you approve of these changes, we can go ahead and merge this PR. We will need to merge this PR before the flowsa v1.3.1 release because there were changes to FBAs and we need access to those FBAs to test the FBS in GitHub Actions

File: esupy/processed_data_mgmt.py [hunk 2]
```diff
@@ -88,7 +89,9 @@ def download_from_remote(file_meta, paths, **kwargs):
                                           + '/' + subdirectory)
                 file = folder + "/" + f
                 create_paths_if_missing(file)
-                log.info('%s saved to %s', f, folder)
+                log.info('%s downloaded from %sindex.html?prefix=%s/%s and '
```
Comment (bl-young) [inline esupy/processed_data_mgmt.py:92]: would you consider changing this to an f-string for readability?

**Comment by bl-young** [comment]
> What's the ramifications for other packages? Do we need to get data for stewi/fedefl/lciafmt/useeio/etc moved over too?

Depending on your thoughts here, I do wonder if we should for now set the `Path.remote_path` [directly in flowsa](https://github.com/USEPA/flowsa/blob/6f0e615a9e4a37690554b7e4341a1361032cdb9d/flowsa/settings.py#L28-L30) to the new link:
e.g. here:
```
"Paths()" are a class defined in esupy
paths = Paths()
paths.local_path = os.path.realpath(paths.local_path + "/flowsa")
```

**Comment by catherinebirney** [comment]
> What's the ramifications for other packages? Do we need to get data for stewi/fedefl/lciafmt/useeio/etc moved over too?

Vincent's team moved all the data from those files to the new url, see [here](https://dmap-data-commons-ord.s3.amazonaws.com/index.html?prefix=). FLOWSA is the only package we had new data for since the team changed urls. Any package that calls on esupy should call data correctly. useeior/R packages will have to be updated separately to use the new urls - but they will continue working for now because the original data commons url has not been discontinued yet

**Comment by bl-young** [comment]
Ok great - then no need for my suggestion

**Pull request merged**